src/app/src/MeetTheFish.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Header = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+`;
+
+const Text = styled.div`
```

Comment:
Can we set this to default to `p` instead of `div`? It's more semantic and will help with styling later.